packages/commons/fs-utils/package.json

```diff
@@ -30,7 +30,7 @@
     "@fern-api/core-utils": "workspace:*",
     "glob": "^8.0.3",
     "glob-promise": "^5.0.0",
-    "json-stream-stringify": "^3.0.1",
+    "json-stream-stringify": "^3.1.4",
```

Comment:
Upgrading this dependency provides a _huge_ performance increase. Before this, serializing our large spec to the IR would take 3-4 minutes, and after this dependency upgrade that is now down to seconds.